test/feature/features/coordinator.feature

```diff
@@ -441,37 +447,6 @@ Feature: Coordinator test
     context deadline exceeded
     """
 
-  Scenario: Coordinator can restart
```

Comment:
At least for now we cannot restart just coordinator, because it's now a part of a router, and not a separate service. There is a scenario for restarting router down below